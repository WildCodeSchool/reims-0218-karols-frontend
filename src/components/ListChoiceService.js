import React from "react"
// Component
import ChoiceService from "./ChoiceService"
import { Row, Col } from "reactstrap"
import CardSelected from "./CardSelected"
import CardHover from "./CardHover"

export default class Service extends React.Component {
  render() {
    return (
      <Row>
        <Col lg="4" className="text-center mb-3">
          <ChoiceService
            name="Préparation"
            description="Description Préparation"
            image="https://images.unsplash.com/photo-1481501940778-c8bb63e376c5?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=842a2f0d2f40bb7bb59bbdfd78289a0f&auto=format&fit=crop&w=1050&q=80"
          />
        </Col>
        <Col lg="4" className="text-center">
          <CardHover
            name="Table + Préparation"
            description="Description Table & Préparation"
            image="https://images.unsplash.com/photo-1466201570621-6dee7c8dbe11?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=0eda6a341afd70d53eaca3b17e674fab&auto=format&fit=crop&w=2250&q=80"
          />
        </Col>
        <Col lg="4" className="text-center">
          <CardSelected
            name="Table"
            description="Description Table"
            image="https://images.unsplash.com/uploads/1412198485051133af17f/5049dacb?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=dddba89812ed0325b966bf07aba683e3&auto=format&fit=crop&w=1050&q=80"
          />
        </Col>
      </Row>
    )
  }
}
