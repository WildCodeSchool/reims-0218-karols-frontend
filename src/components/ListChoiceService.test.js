import React from "react"
import ReactDOM from "react-dom"
import ListChoiceService from "./ListChoiceService"
import renderer from "react-test-renderer"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<ListChoiceService />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders correctly", () => {
  const tree = renderer.create(<ListChoiceService />).toJSON()
  expect(tree).toMatchSnapshot()
})
