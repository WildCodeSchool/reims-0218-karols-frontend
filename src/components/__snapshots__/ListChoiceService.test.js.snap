// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div>
    <div
      className="card"
    >
      <div
        className="card-body"
      >
        <h5
          className="card-title"
        >
           
          Préparation
           
        </h5>
      </div>
      <a
        href=""
      >
        <img
          alt="Name of service"
          className="img-fluid"
          height="250"
          src="http://via.placeholder.com/250x250"
          width="250"
        />
      </a>
      <div
        className="card-body"
      >
        <p
          className="card-text"
        >
           
          Description Préparation
           
        </p>
      </div>
      <div
        className="card-footer"
      >
        <button
          className="btn btn-secondary"
          onClick={[Function]}
          type={undefined}
        >
           Choisir ce service 
        </button>
      </div>
    </div>
  </div>
  <div>
    <div
      className="card"
    >
      <div
        className="card-body"
      >
        <h5
          className="card-title"
        >
           
          Table
           
        </h5>
      </div>
      <a
        href=""
      >
        <img
          alt="Name of service"
          className="img-fluid"
          height="250"
          src="http://via.placeholder.com/250x250"
          width="250"
        />
      </a>
      <div
        className="card-body"
      >
        <p
          className="card-text"
        >
           
          Description Table
           
        </p>
      </div>
      <div
        className="card-footer"
      >
        <button
          className="btn btn-secondary"
          onClick={[Function]}
          type={undefined}
        >
           Choisir ce service 
        </button>
      </div>
    </div>
  </div>
  <div>
    <div
      className="card"
    >
      <div
        className="card-body"
      >
        <h5
          className="card-title"
        >
           
          Table + Préparation
           
        </h5>
      </div>
      <a
        href=""
      >
        <img
          alt="Name of service"
          className="img-fluid"
          height="250"
          src="http://via.placeholder.com/250x250"
          width="250"
        />
      </a>
      <div
        className="card-body"
      >
        <p
          className="card-text"
        >
           
          Description Table & Préparation
           
        </p>
      </div>
      <div
        className="card-footer"
      >
        <button
          className="btn btn-secondary"
          onClick={[Function]}
          type={undefined}
        >
           Choisir ce service 
        </button>
      </div>
    </div>
  </div>
</div>
`;
